public/js/main-template.js

```diff
@@ -12,93 +12,124 @@ $(document).ready(function () {
+
+        }
+        if (event.currentTarget.classList.contains("LeftPannelIcon1")) {
+            event.currentTarget.classList.toggle("Active");
+            event.currentTarget.parentElement.classList.toggle("AsideBarLeftActive");
+        }
     }
-    if (!$(".AsideBarInstallRight").hasClass("RightPanel2Active")) {
-        addClasses([rightMenuBtn], "display-block");
-        removeClasses([rightMenuBtn], "display-none");
+    if (page == "install") {
+        document.getElementsByClassName("LeftPannelIcon1")[0].classList.add("Active");
+        document.getElementsByClassName("AsideBarLeft")[0].classList.add("AsideBarLeftActive");
+        document.getElementsByClassName("PannelIcon2")[0].classList.add("Active");
+        document.getElementsByClassName("AsideBarRight")[0].classList.add("RightPanel2Active");
+        document.getElementsByClassName("RightPanel2")[0].classList.add("Active");
+        document.getElementsByClassName("RightPanel1")[0].classList.remove("Active");
+        document.getElementsByClassName("AsideBarRight")[0].classList.remove("RightPanel1Active");
+        document.getElementsByClassName("PannelIcon")[0].classList.remove("Active");
+
     }
-    else {
-        removeClasses([rightMenuBtn], "display-block");
-        addClasses([rightMenuBtn], "display-none");
+    if (page == "field") {
+        document.getElementsByClassName("LeftPannelIcon1")[0].classList.remove("Active");
+        document.getElementsByClassName("AsideBarLeft")[0].classList.remove("AsideBarLeftActive");
+        document.getElementsByClassName("PannelIcon2")[0].classList.remove("Active");
+        document.getElementsByClassName("AsideBarRight")[0].classList.remove("RightPanel2Active");
+        document.getElementsByClassName("RightPanel2")[0].classList.remove("Active");
+        document.getElementsByClassName("RightPanel1")[0].classList.add("Active");
+        document.getElementsByClassName("AsideBarRight")[0].classList.add("RightPanel1Active");
+        document.getElementsByClassName("PannelIcon")[0].classList.add("Active");
+
     }
 }
 
-function toggleInstallMenu() {
-    $(".LeftPannelIcon1").toggleClass("Active", { duration: 3000 });
-    removeClasses([leftInstallMenuBtn], "display-none");
-    addClasses([leftInstallMenuBtn], "display-block");
-    removeClasses([rightInstallMenuBtn], "display-none");
-    addClasses([rightInstallMenuBtn], "display-block");
-    $(".AsideBarLeft").toggleClass("AsideBarLeftActive");
 
-    // $(".ContentWrapper").removeClass("ChangeFlex");
+function filterPopup(e) {
+    $('.FilterModal').toggleClass('is-visible').animate({ top: 0 });
+
+}
+function modalPopup(e) {
+    $('.StepInstructionModal').toggleClass('is-visible').animate({ top: 0 });
+
+}
+
+function collpaseButton(event) {
+    if (event.currentTarget.classList.value == "collpase-btn") {
+        event.currentTarget.parentElement.parentElement.parentElement.classList.toggle("active")

```

Comment:
i think if you will give unique id to an element then you can access like event.getelementbyid('unique-id').classList.toggel(); this way of chaining is not good as this is very long chaining. if not possible then is ok.